app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb

```diff
@@ -335,7 +335,8 @@ def parse_config_pattern(config_pattern)
     def get_host_relationship(node)
       # Assign a physicalserver and host if server already exists and
       # some host match with physical Server's serial number
-      Host.find_by(:service_tag => node.serialNumber)
+      Host.find_by(:service_tag => node.serialNumber) ||
```

Comment:
We shouldn't need to check both the service_tag and the serial_number now, just the serial_number should be enough.

Reply:
serial number does not seem to have the same value in all cases so it is not guaranteed to always match.    VMware and RHEVM do not seem to have the same value for this field. I would like to leave this check in.

Reply:
What values do they have?  Should be the BIOS UUID in both cases and if not we should fix it.

Reply:
If I look at  RHEVM this field this is still a serial number field and we are able to match that to the serial number of the server.      That is what we are matching to find the RHEVM systems.      I think we do want this field to always match the serial number of the server it is sitting on.  The BIOS ID should be captured as a different field in the hardware object.

Reply:
So RHEV has the serial number in `hardware.serial_number`, what do you need the `service_tag` for?